collector/conntrack_linux.go

```diff
@@ -51,14 +51,14 @@ func (c *conntrackCollector) Update(ch chan<- prometheus.Metric) error {
 	value, err := readUintFromFile(procFilePath("sys/net/netfilter/nf_conntrack_count"))
 	if err != nil {
 		// Conntrack probably not loaded into the kernel.
-		return nil
+		return err
```

Comment:
I think these might want to be similar to our other collectors like this:

```suggestion
    if errors.Is(err, os.ErrNotExist) {
      level.Debug(c.logger).Log("msg", "conntrack probably not loaded")
      return ErrNoData
    }
```

Similar for the other switches from `nil` to `err`.

Reply:
Thanks for the review! I updated for *conntrack* collector, though haven't changed for *pressure* and *rapl* collectors, since for both of them, the function generating the errors return some custom errors rather than os.ErrNotExist. os.ErrNotExist will most likely always evaluate to false for them, so perhaps it makes sense to return either ErrNoData or the error directly?